themes/_base.sass

```diff
@@ -0,0 +1,11 @@
+@mixin component($component)
+  .ais-#{$component}
+    @content
+
+@mixin modifier($modifier)
+  &--#{$modifier}
+    @content
+
+@mixin element($element)
+  &__#{$element}
+    @content
```

Comment:
Great. I did not know that syntax.

I'm not sure the end result in Sass will be any more readable than plain CSS, though :)